coreneuron/sim/multicore.hpp

```diff
@@ -129,8 +129,9 @@ struct NrnThread: public MemoryManaged {
     int _net_send_buffer_cnt = 0;
     int* _net_send_buffer = nullptr;
 
-    int* _watch_types = nullptr;                   /* nullptr or 0 terminated array of integers */
-    void* mapping = nullptr;                       /* section to segment mapping information */
+    int* _watch_types = nullptr; /* nullptr or 0 terminated array of integers */
+    void* mapping = nullptr;     /* section to segment mapping information */
+    void* alu_ = nullptr;        /* report to ALU (values of the current summation */
```

Comment:
Can we forward declare `struct ALUMapping;` and avoid this use of `void*`?

It would be nicer to use `std::unique_ptr<ALUMapping>` and remove some explicit `new`/`delete`, but that might take a little more reorganisation(?) to make the full type visible.